ui/components/component-library/base-avatar/base-avatar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from '../../ui/box/box';
+
+function BaseAvatar(props) {
+  const { size } = props;
+  return (
+    <Box
+      className={classnames('base-avatar', ` --size-${size}`)}
+      data-testid="base-avatar "
```

Comment:
Data ids should be able to be passed as props, I know this allows for all types to be passed in but I think there is an argument for spreading all props down to the root element. This will need to be updated in the Box component